Jenkinsfile

```diff
@@ -1,49 +1,50 @@
-@Library('defra-library@0.0.3')
+@Library('defra-library@psd-190-support-default-image-names')
 import uk.gov.defra.ffc.DefraUtils
 def defraUtils = new DefraUtils()
 
 def registry = '562955126301.dkr.ecr.eu-west-2.amazonaws.com'
 def regCredsId = 'ecr:eu-west-2:ecr-user'
 def kubeCredsId = 'awskubeconfig002'
 def ingressServer = 'ffc.aws-int.defra.cloud'
-def imageName = 'ffc-demo-web'
 def repoName = 'ffc-demo-web'
-def pr = ''
-def mergedPrNo = ''
+def serviceName = 'app'
 def containerTag = ''
+def mergedPrNo = ''
+def pr = ''
+def projectName = ''
 
 node {
   checkout scm
   try {
     stage('Set PR, and containerTag variables') {
-      (pr, containerTag, mergedPrNo) = defraUtils.getVariables(repoName)
+      (pr, containerTag, mergedPrNo, projectName) = defraUtils.getVariables(repoName)
```

Comment:
`projectName` is now the concatenation of repo name with PR and build numbers, which is generated once in the shared library rather than concatenating multiple variables in several places.

Reply:
I like the change to a project name setting, as it removes the ugly PR_NO that is being passed around.